src/element/mutateElement.ts

```diff
@@ -114,3 +114,14 @@ export const newElementWith = <TElement extends ExcalidrawElement>(
     versionNonce: randomInteger(),
   };
 };
+
+/**
+ * Mutates element and updates `version` & `versionNonce`.
+ *
+ * NOTE: does not trigger re-render.
+ */
+export const bumpVersion = (element: Mutable<ExcalidrawElement>) => {
+  element.version = element.version + 1;
+  element.versionNonce = randomInteger();
+  return element;
+};
```

Comment:
slightly controversial: mutates version and doesn't notify the renderer. This is done because of perf because we're potentially updating this in a loop when changing the z indices of many elements. And lastly, it's done as part of an action that notifies the renderer on its own.